### PR TITLE
Add options

### PR DESCRIPTION
**Pull request by ZewdieMc**
### Hello @Besufekad-HAZ  and @byiringiroscar , 
In this PR, I implemented the following

- [ ] List all games
- [ ] List all authors
- [ ] add a game
- [ ] modified the `options.rb` to only contain `[0..9] `options
- [ ] Add constructor to the `Main` class in `main.rb` file.
- [ ] Moved the code which was in the constructor of the `Choice` class to a new method called `run`.

### CODE REVIEW

File: choice.rb [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
-    11 => :add_movie,
-    12 => :add_game,
+    3 => :list_games,
+    4 => :list_genres,
+    5 => :list_labels,
+    6 => :list_authors,
+    7 => :add_book,
+    8 => :add_music_album,
+    9 => :add_game,
     0 => :exit
```
Comment (Besufekad-HAZ) [inline choice.rb:15]: @ZewdieMc you did a great job minimizing the options and removing unnecessary ones. ⚡

File: item.rb [hunk 2]
```diff
@@ -3,16 +3,16 @@
...
 
 class Item
-  attr_accessor :genre, :author, :source, :label, :publish_date
+  attr_accessor :genre, :author, :label, :publish_date
   attr_reader :id, :archived
 
   def initialize(publish_date, archived)
     @id = SecureRandom.random_number(1000)
-    @publish_date = Date.parse(publish_date)
+    @publish_date = publish_date
```
Comment (Besufekad-HAZ) [inline item.rb:11]: @ZewdieMc Nice call on identifying and correcting the date parsing issue that displays an incorrect date message.

File: game/game_ui.rb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    print 'Enter game author First Name: '
+    first_name = gets.chomp
+    print 'Enter game author Last Name: '
+    last_name = gets.chomp
+    author = Author.new(first_name, last_name)
+    author.add_item(game)
+    @games.push(game)
+    puts 'Game created successfully.'
+  end
+
```
Comment (Besufekad-HAZ) [inline game/game_ui.rb:29]: @ZewdieMc you did great.
- One optional suggestion can be to add error handling for cases where the user enters invalid input, such as an invalid date or an invalid input for the multiplayer option.

**Review by Besufekad-HAZ — APPROVED** [review]
## Hello @ZewdieMc    👏🏽, 

clapping

# ✔️✔️✔️   Status: APPROVED 🏆 🎉 🎊 

Your pull request is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good Implementation of the options 👏🏽 
- Used correct git flow 🥇 
- All added tests are passing👌🏽
- All linters are passing ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@Besufekad-HAZ ) in your question so I can receive the notification.

------

File: item.rb (hunk 2 above)
Comment (ZewdieMc) [reply item.rb:11]: Thank you @Besufekad-HAZ 
It was initially giving an error, then I figured out the `Date.parse` method was not necessary.

**Pull request merged**